src/Algebra/Solver/Monoid/Tactic.agda

```diff
@@ -0,0 +1,52 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- A solver for equations over monoids
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Bundles using (Monoid)
+import Algebra.Solver.Monoid.Expression as Expression
+
+module Algebra.Solver.Monoid.Tactic {a c ℓ} (M : Monoid c ℓ)
```

Comment:
Very minor, but I would prefer we didn't use the term `Tactic` outside of the top-level tactic folder as this does not export a tactic macro per se. I'm open to other suggestions, but we could call this `Implementation` or proof?

Reply:
`Solver`?